docs/index.rst

```diff
@@ -58,3 +58,5 @@ In a terminal, run the following:
    * :ref:`genindex`
    * :ref:`modindex`
    * :ref:`search`
+
+.. image:: ./images/installation.gif
```

Comment:
this should be at the end of quickstart section, not at the end of the file.

Reply:
I did it. 💯